packages/datetime/src/components/date-input/dateInput.tsx

```diff
@@ -245,6 +246,7 @@ export const DateInput: React.FC<DateInputProps> = React.memo(function _DateInpu
 
     const inputRef = React.useRef<HTMLInputElement | null>(null);
     const popoverContentRef = React.useRef<HTMLDivElement | null>(null);
+    const popoverId = uniqueId("date-picker");
```

Comment:
can you use `Utils.uniqueId()` instead? `import { Utils} from "@blueprintjs/core"`